script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -835,21 +856,23 @@ There's no mistaking it. That was the sound of a blade.
 [sha:c1677caf81fc048e3eae27ad05ef74b976e34afc]{
 -- Page 859, Offset 7944.
 -- あのコートの下に<長物|ナガモノ>が隠されている。エレベーター前で女を切り裂いたのもソレだろう。
-There's a long object hidden under that coat. It must be the one that he used to slash the woman in front of the elevator.
+There's a long object hidden under that coat.
+ It must be the same thing he used to slash the woman in front of the elevator.
 }
 [sha:b3fae28b5b701599ff3006d0135122e0581e5a2f]{
 -- Page 860, Offset 7945.
 -- 男の足が動く。
-The man moves his legs.
+The man begins to move.
 }
 [sha:92c4f761eeb3ee37de530e2f14ae06f74683882f]{
 -- Page 860, Offset 7946.
 -- 刃物をコートの下に隠したまま、アルクェイドに向かって一歩踏み込もうとする。
-With the blade still hidden under his coat, he tries to take a step toward Arcueid.
+With the blade still hidden under his coat, he goes to take a step toward Arcueid.
 }
 [sha:0b7e306dda0e1ace0aa9dde9b8911345b3ec13c4]{
 -- Page 861, Offset 7947.
 -- 「“無礼者。豚でなければ<銘|な>を名乗れ。
+// Add text effect on þe olde english?
```

Comment:
we had talked before about using like a gothic or fraktur-esque nazi font for the old english. That might need to be a post-kerning change since I think it might've looked a bit like crap